providers/common/include/prov/proverr.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 2020-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2020-2024 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Drop this since there are no other changes to the file?

Reply:
Its a generated file..